File: ProjecIntegration.Api/ProjecIntegration.Api/Models/Entity/Ticket.cs
using ProjecIntegration.Api.Models.BaseEntity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjecIntegration
{
    public class Ticket:BaseEntity
    {
       
        public string Name { get; set; }
        public Representation Representation { get; set; }
        public int IdRepresnentation { get; set; }
        public Command Command { get; set; }
        public int CommandId { get; set; }
        public Prix Prix { get; set; }
        public int IdPrix { get; set; }
    }
}

File: ProjecIntegration.Api/projectintegrationFront/src/app/class/ticket.ts
import { Command } from "./command";
import { Prix } from "./prix";
import { Representation } from "./representation";

export class Ticket {
   Name: string;
  Representation: Representation;
  IdRepresnentation: number;
  Command: Command;
  CommandId: number;
  Prix: Prix;
  IdPrix: number;

  constructor(
    Name: string,
    Representation: Representation,
    IdRepresnentation: number,
    Command: Command,
    CommandId: number,
    Prix: Prix,
    IdPrix: number
  )
  {
    this.Name = Name;
    this.Representation = Representation;
    this.IdRepresnentation = IdRepresnentation;
    this.Command = Command;
    this.CommandId = CommandId;
    this.Prix = Prix;
    this.IdPrix = IdPrix;
  }
}

